services/tenant-ui/frontend/src/components/innkeeper/tenants/Tenants.vue

```diff
@@ -33,7 +33,10 @@
       <Column :expander="true" header-style="width: 3rem" />
       <Column :sortable="false" :header="$t('common.actions')">
         <template #body="{ data }">
-          <EditConfig :tenant="data" />
+          <div class="container">
+            <EditConfig :tenant="data" />
+            <DeleteTenant :tenant="data" />
```

Comment:
@gurjmatharu can we block the delete if it's the Innkeeper Tenant? I think the only way to reliably know is to check `tenant_name` is `traction_innkeeper` since the IDs vary in each environment. Should be ok since that's the convention for Traction anyways.

While you're at it we can also disable the EditConfig gear there too since there's no need for Innkeeper issuance stuff. Been meaning to do that for a while.

![image](https://github.com/bcgov/traction/assets/17445138/8421905d-2cf2-45da-ba15-d676e2324059)